text/0000-default-serializers-and-adapters.md

```diff
@@ -0,0 +1,53 @@
+
+In addition to removing use of initializer injection, this takes a significant step toward simplifying the mental model for how to determine what adapter/serializer is in use. Removing the defaults forces app developers to be more cognizant about the type of application level concerns vs model-specific concerns; they will now need to explicitly define and use specific adapters/serializers. After this deprecation RFC lands, apps will always use an adapter/serializer explicitly put into your application and the rule will be "the adapter matching modelName falling back to application".
+
+## Detailed design
+The injection of `-default` and `-json-api` serializers will be removed in the next major version (4.0). Since this changes some core assumptions we will deprecate the reliance on the existence of the defaults. All deprecation warnings will only be shown in Dev mode.
+
+##### deprecate -default serializer fallback in Store.serializerFor
+A deprecation warning will be shown when [no model, application or adapter serializer](https://github.com/emberjs/data/blob/67affb0eca7048a1a0edc856af46d1305cd1fc1d/packages/store/addon/-private/system/store.ts#L2909) has specified and the default must be used.
+
+##### deprecate Adapter.serializer and Adapter.defaultSerializer fallbacks
+A deprecation warning will be shown when accessing the [adapter's defaultSerializer](https://github.com/emberjs/data/blob/67affb0eca7048a1a0edc856af46d1305cd1fc1d/packages/store/addon/-private/system/store.ts#L2896). This will be distinct from the warning about using the application level default.
+
+##### deprecate Store.defaultAdapter (-json-api) and the -json-api adapter fallback behavior
+A deprecation warning will be shown when the [defaultAdapter](https://github.com/emberjs/data/blob/b0cf3225662bfb806cd0c02b55b763e37a319b32/packages/store/addon/-private/system/store.ts#L309) is accessed.
+
+##### deprecate Model.toJSON
+A deprecation warning will be shown when toJSON is called since it uses a serializer to create a JSON representation of [the model](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers). Users may call record.serialize() or implement their own toJSON instead.
+
+## How we teach this
+
+Today we have extensive documentation about creating custom serializers, but we will need to update the guides to specify the desired serializer in [app/serializers/application.js](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
+
+The deprecation guide app will be updated with examples showing how to
+migrate away from relying on the defaults.
```

Comment:
Will this include updates to any blueprints?  For example adding a default adapter with `ember new`?

Reply:
That sounds like a great idea, but I'm not sure when it should be mentioned...the RFC is about deprecation, but I think updating the blueprints for the deprecation guide is a good idea. @runspired what are your thoughts?

Reply:
I think that would be a different RFC if at all